### PR TITLE
chore(logs): The log format starts with a capital letter in internal/dataplane

### PR DESCRIPTION
**Pull request by pangruoran**
**What this PR does / why we need it**:

**Which issue this PR fixes**:

fixes: #3708 
**Special notes for your reviewer**:

This PR more about `internal/dataplane`

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [x] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -211,6 +211,8 @@ Adding a new version? You'll need three changes:
   [#5038](https://github.com/Kong/kubernetes-ingress-controller/pull/5038)
   [#5049](https://github.com/Kong/kubernetes-ingress-controller/pull/5049)
   [#5050](https://github.com/Kong/kubernetes-ingress-controller/pull/5050)
+  [#5065](https://github.com/Kong/kubernetes-ingress-controller/pull/5065)
+
```
Comment (pmalek) [inline CHANGELOG.md:215]:
```suggestion
```

Comment (pangruoran) [reply CHANGELOG.md:215]: Thanks!

**Comment by pangruoran** [comment]
This will be the last PR for this part of the modification. (it will close #3708)
I hope it can be merged before the release of version 3.0.
Thanks

**Review by pmalek — DISMISSED** [review]

**Comment by pangruoran** [comment]
Sorry, I missed a modification. It has been updated.

**Review by pmalek — APPROVED** [review]

File: internal/dataplane/kong_client.go [hunk 2]
```diff
@@ -395,17 +395,17 @@ func (c *KongClient) Update(ctx context.Context) error {
 		}
 	}
 
-	c.logger.V(util.DebugLevel).Info("parsing kubernetes objects into data-plane configuration")
+	c.logger.V(util.DebugLevel).Info("Parsing kubernetes objects into data-plane configuration")
```
Comment (randmonkey) [inline internal/dataplane/kong_client.go:398]: https://github.com/Kong/kubernetes-ingress-controller/blob/13da44018b05d518a01f49bc10320f1c4b710871/internal/dataplane/kong_client.go#L185 
Here should be modified.

**Review by randmonkey — CHANGES_REQUESTED** [review]
`kubernetes-ingress-controller/test/integration/translation_failures_test.go` should be updated to match the modified translation failure reasons.

**Review by randmonkey — APPROVED** [review]

**Pull request merged**